README.md

```diff
@@ -17,34 +17,74 @@ Example application for handling ShipEngine Elements authentication token reques
 
 1. Install dependencies
 
+Install with npm
+
 ```bash
 npm install
 ```
 
+Installl with yarn
+
 ```bash
 yarn
 ```
 
 2a. Start the application
 
+Start with npm
+
 ```bash
 npm start
 ```
 
+Start with yarn
+
 ```bash
 yarn start
 ```
 
 2b. Start application for development
 
+Start with npm in dev mode (watch mode)
+
 ```bash
 npm dev
 ```
 
+Start with yarn in dev mode (watch mode)
+
 ```bash
 yarn dev
 ```
 
+3. Request a token from the application
```

Comment:
A token can be generated without passing the tenant ID? and what would be the reason to exclude it?